Sources/NIO/Bootstrap.swift

```diff
@@ -175,6 +174,18 @@ public final class ServerBootstrap {
         self._serverChannelOptions.append(key: option, value: value)
         return self
     }
+    
+    /// Specifies some `ChannelOption`s to be applied to the `ServerSocketChannel`.
+    /// - See: serverChannelOption
+    /// - Parameter options: List of shorthand options to apply.
+    /// - Returns: The updated server bootstrap (`self` being mutated)
+    @inlinable
+    public func serverOptions(_ options: [ServerOption]) -> Self {
```

Comment:
there are two API discrepancies here:

1. it's `serverOptions` vs. `childChannelOptions` (one has `...Channel...` the other one doesn't)
2. `ServerOption` seems `ServerBootstrap` specific (what about `NIOTSListenerBootstrap`) vs `NIOTCPShorthandOption` which is more generic

Reply:
OK, I have pulled all the option types out to be top level classes so they can be reused in any other implementations which might exist in the future.  Added Channel to name.